packages/fast-components-react-base/src/checkbox/checkbox.tsx

```diff
@@ -92,7 +91,7 @@ class Checkbox extends Foundation<ICheckboxHandledProps & ICheckboxManagedClasse
                 />
                 <span className={get(this.props, "managedClasses.checkbox_stateIndicator")} />
                 <span className={get(this.props, "managedClasses.checkbox_label")}>
-                    {this.props.text ? this.props.text : null}
+                    {this.props.children}
```

Comment:
Wondering if we should render the label at all if we do not have children?

Reply:
We shouldn't, but I don't think this would be breaking and could go in prior to v3.0

Reply:
@chrisdholt we should probably check the rest of the components as well - here's a [task](https://github.com/Microsoft/fast-dna/issues/948) for it.